script/Ciel/Day 13/13_01_02_CIEL13_7F.txt

```diff
@@ -1,42 +1,46 @@
 [sha:e1bc864eb39c160235fd757a9c2c3a323bf0e067]{
 -- Page 10014, Offset 36448.
 -- 先輩のアパートに着くまで、人目に付く事はなかった。
-We've managed to make it back to the apartment without being seen.
+We've managed to make it back to Senpai's apartment without anyone seeing us.
 }
 [sha:e22a3f853743a52b15d6a98de4926532888f4fed]{
 -- Page 10014, Offset 36449.
 -- なんでも最小限の人避けの魔術を使っているとかで、先輩とくっついているかぎり他人に見つかる恐れはないとのコト。
-Senpai said she was using some basic anti-detection magecraft, so no one should be able to see me as long as I stay close to her.
+Apparently she's using some sort of basic anti-detection magecraft,
+ so as long as I stick close to her, I don't need to worry about being spotted.
```

Comment:
I guess I can't come up with anything better than "people repellent".

I do want it to be more clear what's going on rather than it seeming like some kind of stealth-field, you know.